src/server/db.js

```diff
@@ -0,0 +1,60 @@
+var mysql     =    require('mysql');
+
+// URL of the MySQL server. We expect that the DB is listening on the
+// default port (3306).
+//
+const DB_HOST = 'heavenly';
```

Comment:
Should move all these config constants into configuration. Config files are `config/default.json` and `config/production.json`. Example of use:

default.json
```javascript
{ 
    "db": {
        "host": "heavenly"
    }
}
```
db.js:
```javascript
import config from 'config';

const pool = mysql.createPool({
    host: config.db.host
});
```